assets/js/index.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
 require('hof-theme-govuk');
+
+/* eslint-disable no-undef */
+document.getElementById('add-image-button').classList.remove('visuallyhidden');
```

Comment:
eslint complains about document not being defined.  Therefore, I've disabled the rule here

Reply:
Don't disable the rule. Fix the config. 

You need to ensure that code in `./assets` is linted with the environment set to browser.